benjamindannegard/ Updated bootloader update text (#127)
arduino/docs-content: content/hardware/04.pro/boards/portenta-h7/tutorials/por-openmv-gs/content.md

Before you can start programming OpenMV scripts for the Portenta you need to download and install the OpenMV IDE.

> ***IMPORTANT: Please make sure to update the bootloader to the most recent version to benefit from the latest improvements. As explained in the "Flashing the OpenMV Firmware" section before you proceed with the next step of this tutorial!***

Reviewer: ```suggestion
***IMPORTANT: Please make sure to update the bootloader to the most recent version to benefit from the latest improvements. Follow [these steps](/tutorials/portenta-h7/updating-the-bootloader) before you proceed with the next step of this tutorial.***

```
